wiki/Tournaments/MCLT/4K2025/en.md

```diff
@@ -0,0 +1,386 @@
+5. This tournament is a 1v1 double-elimination competition for the osu!mania 4K game mode.
+6. The score system used will be ScoreV2, and the version used will be stable, not osu!lazer.
+7. No tournament staff, except designers, streamers, and commentators, may participate as players.
+8. Players must adhere to the [osu! community rules](/wiki/Rules) at all times. Any violation of these rules may result in expulsion at the discretion of the tournament hosts.
+9. Players are allowed to adjust visual settings to alter background opacity or disable beatmap elements, such as storyboards and skins.
+10. Skin elements must not be used to alter key gameplay mechanics inappropriately.
+11. The tournament hosts reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Mappool information
+
+1. Every stage will have its own mappool.
+2. Each mappool consists of a fixed amount of maps, which will all be played under Free Mod conditions.
+3. The mappool sizes are as follows:
+   - Qualifiers: 5 beatmaps
+   - Round of 16: 13 beatmaps
+   - Quarterfinals and Semifinals: 15 beatmaps
+   - Finals and Grand Finals: 17 beatmaps
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for all beatmaps are [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Fade In](/wiki/Gameplay/Game_modifier/Fade_In), [Flashlight](/wiki/Gameplay/Game_modifier/Flashlight), and [Mirror](/wiki/Gameplay/Game_modifier/Mirror).
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Qualifiers
+
+1. A total of 8 lobbies will be available at different times. Custom lobbies can be requested if a player cannot participate in any of the designated lobbies.
+2. Players will be invited 10 minutes before the start of each lobby. It is not necessary to join before this period, but players must be present at the scheduled time.
+3. If a player does not show up within 5 minutes of the lobby's start time, they will be moved to another lobby.
+4. Each lobby will consist of 5 consecutive maps, with a 2-minute break after each map. Once all maps are completed, there will be a 5-minute break where players can retry their maps.
+5. The seeding method used will be Average Rank, based on the average ranking obtained in each map.
+
+### After Qualifiers
+
+1. On Sunday of the qualifiers week, at a previously announced time, the qualifier results and designated matches will be published.
+2. If players qualify, they will advance to the double-elimination bracket stage, meaning that losing your first match does not eliminate you from the tournament. Starting from Quarterfinals, if you are in the loser bracket, you will have to play two matches per week. If you lose in the losers bracket, players will get eliminated from the tournament.
+
+### Matches
+
+1. A room will be created, and players will be invited 15 minutes before the match, with the format: `MCLT 4K 2025: {(Player 1)} vs {(Player 2)}`
```

Comment:
```suggestion
1. A room will be created, and players will be invited 15 minutes before the match, with the format: `MCLT4K2025: {(Player 1)} vs {(Player 2)}`
```
I randomlly check some mplink, those not following the format with space.
https://osu.ppy.sh/community/matches/117205295
https://osu.ppy.sh/community/matches/117188342
https://osu.ppy.sh/community/matches/117190863